initramfs/init.sh

```diff
@@ -421,8 +443,10 @@ if [ "${RESCUE_SHELL}" = "yes" ]; then
     rescue_shell
 fi
 
+set_display_splash
```

Comment:
You are calling the set_display_splash twice

Reply:
Yes, it is on purpose. In case of setting/changing the Article Number, the user will see the image immediately after writing the Article Number to the EEPROM.

But for regular boots, calling the set_display_splash before find_and_run_update save some time and display the splash faster.

I think we could let both calls, but if you prefer, I can remove the latest one.